### PR TITLE
taxonomy: translations for ingredients_processing.txt

### PR DESCRIPTION
**Pull request by teolemon**
### What
taxonomy: translations for ingredients_processing.txt

### CODE REVIEW

**Comment by teolemon** [comment]
Errors in the ingredients_processing taxonomy definition:
ERROR - de:würfel already is associated to en:cubes (ingredients_processing) - de:würfel cannot be mapped to entry en:diced
ERROR - hu:kockára-vágva already is a synonym of hu:kocka for entry en:cubes (ingredients_processing) - hu:kockára-vágva cannot be mapped to entry en:diced / hu:kockázott
ERROR - hu:kockára-vágott already is a synonym of hu:kocka for entry en:cubes (ingredients_processing) - hu:kockára-vágott cannot be mapped to entry en:diced / hu:kockázott
ERROR - nl:blokjes already is associated to en:cubes (ingredients_processing) - nl:blokjes cannot be mapped to entry en:diced
ERROR - fi:rouhe already is associated to en:crumb (ingredients_processing) - fi:rouhe cannot be mapped to entry en:grits
ERROR - it:reidratato already is associated to en:rehydrated (ingredients_processing) - it:reidratato cannot be mapped to entry en:reconstituted
ERROR - en:extracted already is a synonym of en:extracted for entry de:extrahiert (ingredients_processing) - en:extracted cannot be mapped to entry en:extract / en:extract
ERROR - hu:préselt already is associated to en:pressed (ingredients_processing) - hu:préselt cannot be mapped to entry en:expeller-pressed
ERROR - bg:сухо-печено already is a synonym of bg:сухо-печено for entry en:dry-baked (ingredients_processing) - bg:сухо-печено cannot be mapped to entry en:dry-roasted / bg:сухо-печен
ERROR - ja:乾燥 already is a synonym of ja:ドライ for entry en:dried (ingredients_processing) - ja:乾燥 cannot be mapped to entry en:dry-roasted / ja:乾燥
ERROR - de:gezuckert already is associated to en:sugared (ingredients_processing) - de:gezuckert cannot be mapped to entry en:crystallized

File: taxonomies/ingredients_processing.txt [hunk 1]
```diff
@@ -273,8 +309,11 @@ sv:hackade
 
 en:cubes,cubed
 ca:cub, cubs
-hr:kockice
+de:Würfel, gewürfelt, gewürfelte, in Würfel geschnitten, in Würfel
```
Comment (teolemon) [inline taxonomies/ingredients_processing.txt:312]:
```suggestion
# de:Würfel, gewürfelt, gewürfelte, in Würfel geschnitten, in Würfel
```

File: taxonomies/ingredients_processing.txt [hunk 2]
```diff
@@ -273,8 +309,11 @@ sv:hackade
...
 es:cubo,cubos
+hr:kockice
+hu:kocka, kockák, kockára vágva, kockára vágott
```
Comment (teolemon) [inline taxonomies/ingredients_processing.txt:315]:
```suggestion
# hu:kocka, kockák, kockára vágva, kockára vágott
```

File: taxonomies/ingredients_processing.txt [hunk 3]
```diff
@@ -273,8 +309,11 @@ sv:hackade
...
+nl:blokjes, in blokjes, blokje
```
Comment (teolemon) [inline taxonomies/ingredients_processing.txt:316]:
```suggestion
# nl:blokjes, in blokjes, blokje
```

File: taxonomies/ingredients_processing.txt [hunk 4]
```diff
@@ -1767,13 +2002,38 @@ en:semi candied, semi-candied
 fr:semi-confit, semi confit
 
 #en:crystallized
+en:crystallized
+bg:кристализиран, кристализирана, кристализирани, кристализирани
+ca:cristal·litzat, cristal·litzada, cristal·litzats, cristal·litzades
+de:gezuckert, gezuckerte, gezuckerter, gezuckertes
```
Comment (teolemon) [inline taxonomies/ingredients_processing.txt:2008]:
```suggestion
```

File: taxonomies/ingredients_processing.txt [hunk 5]
```diff
@@ -1254,15 +1427,42 @@ sv:nypressad
...
+pt:espremido naturalmente, espremida naturalmente, espremidos naturalmente, espremidas naturalmente
+sv:naturligt pressad, naturligt pressade, naturligt pressat, naturligt pressade
 
 #<en:pressed
 en:expeller pressed
+bg:екстракция с преса
+de:Schneckenpressung
+es:prensado por extrusión
+fr:pressé à l'expulseur
+hu:préselt
```
Comment (teolemon) [inline taxonomies/ingredients_processing.txt:1460]:
```suggestion
```

File: taxonomies/ingredients_processing.txt [hunk 6]
```diff
@@ -1550,7 +1767,18 @@ sv:rostad, rostade, bakad
 
 #<en:roasted
 en:dry roasted
-ja:焙煎
+bg:сухо печен, сухо печена, сухо печено, сухо печени
```
Comment (teolemon) [inline taxonomies/ingredients_processing.txt:1770]:
```suggestion
```

File: taxonomies/ingredients_processing.txt [hunk 7]
```diff
@@ -1086,7 +1233,17 @@ de:sprühgetrocknet, sprühgetrocknete, sprühgetrockneter, sprühgetrocknetes
 
 #<en:dried
 en:dry baked
+bg:сухо печено
```
Comment (teolemon) [inline taxonomies/ingredients_processing.txt:1236]:
```suggestion
```

File: taxonomies/ingredients_processing.txt [hunk 8]
```diff
@@ -1550,7 +1766,17 @@ sv:rostad, rostade, bakad
...
 #<en:roasted
 en:dry roasted
-ja:焙煎
+ca:torrat a l'aire, torrats a l'aire, torrada a l'aire, torrades a l'aire
+de:trocken geröstet, trocken geröstete, trocken geröstetes, trocken gerösteter
+es:asado al seco, asada al seco, asados al seco, asadas al seco
+fr:grillé à sec, grillée à sec, grillés à sec, grillées à sec
+hu:szárazon pirított, szárazon pirítottak, szárazon pirított, szárazon pirítottak
+it:arrostito a secco, arrostita a secco, arrostiti a secco, arrostite a secco
+ja:乾燥
```
Comment (teolemon) [inline taxonomies/ingredients_processing.txt:1775]:
```suggestion
ja:焙煎
```

File: taxonomies/ingredients_processing.txt [hunk 9]
```diff
@@ -498,7 +550,25 @@ sv:bitar, i bitar
 # de:false-positive:früchstück
 
 en:crumb, crumbs
+bg:мръсница, мръсници
+ca:mig, miques
+cs:drob, droby
+da:brødkrumme, brødkrummer
+de:Brösel
+es:migaja, migajas
+fi:rouhe, rouheet
```
Comment (teolemon) [inline taxonomies/ingredients_processing.txt:559]:
```suggestion
# fi:rouhe, rouheet
```

File: taxonomies/ingredients_processing.txt [hunk 10]
```diff
@@ -1173,12 +1340,17 @@ en:partially hydrated
 fr:semi-hydraté, semi-hydratée, semi-hydratés, semi-hydratées, partiellement hydraté, partiellement hydratée, partiellement hydratés, partiellement hydratées
 
 en:reconstituted
+bg:реконституиран, реконституирано, реконституирана, реконституирани
 ca:reconstituida
+de:wiederaufbereitet, wiederaufbereitete, wiederaufbereiteter, wiederaufbereitetes
+es:reconstituido, reconstituida, reconstituidos, reconstituidas
 fr:reconstitué, reconstituée, reconstitués, reconstituées
 hr:rekonstituirano
+it:reidratato, reidratata, reidratati, reidratate
```
Comment (teolemon) [inline taxonomies/ingredients_processing.txt:1349]:
```suggestion
```

**Review by benbenben2 — APPROVED** [review]
Good work, one suggestion

**Pull request merged**